x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/perform_rule_upgrade/perform_rule_upgrade_route.ts

```diff
@@ -60,6 +60,7 @@ export const performRuleUpgradeRoute = (router: SecuritySolutionPluginRouter) =>
           const ctx = await context.resolve(['core', 'alerting', 'securitySolution']);
           const soClient = ctx.core.savedObjects.client;
           const rulesClient = ctx.alerting.getRulesClient();
+          const rulesManagementClient = ctx.securitySolution.getRulesManagementClient();
```

Comment:
Do we also need to pass `ml` here?

Reply:
I changed the implemantation here so we don't have to pass `ml` to `getRulesManagementClient`